sql-parser/src/test/java/io/crate/sql/IdentifiersTest.java

```diff
@@ -34,7 +34,7 @@
     public void test_number_of_keywords() {
         // If this test is failing you are introducing a new reserved keyword which is a breaking change.
         // Either add the new term to `nonReserved` in `SqlBase.4g` or add a breaking changes entry and adapt this test.
-        assertThat(Identifiers.KEYWORDS.size(), is(95));
+        assertThat(Identifiers.KEYWORDS.size(), is(96));
```

Comment:
If `RETUNING` becomes, the [lexical structure](https://crate.io/docs/crate/reference/en/latest/sql/general/lexical-structure.html#key-words-and-identifiers) has to be adjusted as well.

Reply:
Good point,  but `RETURNING` will become a non-reserved keyword.